sumo-api/sumo/metrics/voxel_evaluator.py

```diff
@@ -0,0 +1,141 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+
+Algorithm class: Evaluate a voxel track submission
+"""
+
+from sumo.metrics.evaluator import Evaluator
+from sumo.metrics.mesh_evaluator import sample_element
+import sumo.metrics.utils as utils
+
+class VoxelEvaluator(Evaluator):
+    """
+    Algorithm to evaluate a submission for the mesh track.
+    """
+
+    def __init__(self, submission, ground_truth, settings=None):
+        """
+        Constructor.  Computes similarity between all elements in the
+        submission and ground_truth and also computes 
+        data association caches. 
+
+        Inputs:
+        submission (ProjectScene) - Submitted scene to be evaluated
+        ground_truth (ProjectScene) - The ground truth scene
+        settings (EasyDict) - configuration for the evaluator.  See
+        Evaluator.py for recognized keys and values.   
+        """
+
+
+        # TODO: not needed anymore.  Use this as a template for converter
```

Comment:
If not needed, remove

Reply:
We will use this in the code shortly.  Prefer to leave it in for now.